chains/neoxt4/logo_x.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="2010" height="2010" viewBox="0 0 2010 2010" fill="none" xmlns="http://www.w3.org/2000/svg">
+<mask id="mask0_968_137" style="mask-type:alpha" maskUnits="userSpaceOnUse" x="0" y="0" width="2010" height="2010">
+<rect width="2010" height="2010" fill="#D9D9D9"/>
+</mask>
+<g mask="url(#mask0_968_137)">
+<path d="M1724.36 239.714L1574.2 85.2104C1501.2 10.249 1378.57 16.9123 1314.34 99.3698L880.542 657.416L565.206 253.874C499.302 169.75 366.66 187.241 325.366 286.357L250.286 466.264C227.345 521.236 234.436 584.12 269.056 632.429L576.885 1063.46L304.928 1428.27C264.468 1482.41 262.382 1555.7 298.671 1612.34L380.425 1738.53C439.655 1829.73 570.211 1837.64 639.868 1754.77L875.536 1474.08L1196.71 1907.19C1247.18 1975.07 1339.78 1996.73 1414.86 1957.58L1584.21 1869.71C1680.14 1819.73 1706.84 1694.8 1639.69 1610.26L1217.99 1077.62L1732.28 470.429C1789.43 402.964 1786.09 303.015 1724.36 239.714ZM1658.46 407.961L1104.11 1062.21C1097.44 1070.12 1097.44 1081.36 1103.7 1089.28L1563.77 1669.81C1592.97 1706.88 1581.29 1761.43 1539.58 1783.5L1370.23 1871.37C1337.28 1888.45 1296.82 1879.29 1274.71 1849.3L896.809 1339.56C888.467 1328.32 871.782 1327.9 863.023 1338.73L566.04 1692.72C538.093 1726.03 485.954 1722.7 462.179 1686.47L380.425 1560.28C365.826 1537.8 366.66 1508.23 382.928 1486.57L687.836 1077.62C693.259 1070.12 693.676 1060.13 688.253 1052.21L348.724 576.624C333.708 555.385 330.372 527.899 340.382 504.161L415.462 324.254C427.976 293.853 468.853 288.439 489.291 314.259L863.857 793.179C872.617 804.424 889.301 804.424 898.061 793.179L1391.09 158.506C1419.03 122.275 1472.84 119.36 1504.96 152.259L1655.12 306.763C1681.81 334.665 1683.48 378.393 1658.46 407.961Z" fill="#00E599"/>
+</g>
+</svg>
```

Comment:
ran through svg minify https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="2010" height="2010" fill="none"><mask id="a" width="2010" height="2010" x="0" y="0" maskUnits="userSpaceOnUse" style="mask-type:alpha"><path fill="#D9D9D9" d="M0 0h2010v2010H0z"/></mask><g mask="url(#a)"><path fill="#00E599" d="M1724.36 239.714 1574.2 85.21c-73-74.961-195.63-68.298-259.86 14.16L880.542 657.416 565.206 253.874c-65.904-84.124-198.546-66.633-239.84 32.483l-75.08 179.907c-22.941 54.972-15.85 117.856 18.77 166.165l307.829 431.031-271.957 364.81c-40.46 54.14-42.546 127.43-6.257 184.07l81.754 126.19c59.23 91.2 189.786 99.11 259.443 16.24l235.668-280.69 321.174 433.11c50.47 67.88 143.07 89.54 218.15 50.39l169.35-87.87c95.93-49.98 122.63-174.91 55.48-259.45l-421.7-532.64 514.29-607.191c57.15-67.465 53.81-167.414-7.92-230.715Zm-65.9 168.247-554.35 654.249c-6.67 7.91-6.67 19.15-.41 27.07l460.07 580.53c29.2 37.07 17.52 91.62-24.19 113.69l-169.35 87.87c-32.95 17.08-73.41 7.92-95.52-22.07l-377.901-509.74c-8.342-11.24-25.027-11.66-33.786-.83L566.04 1692.72c-27.947 33.31-80.086 29.98-103.861-6.25l-81.754-126.19c-14.599-22.48-13.765-52.05 2.503-73.71l304.908-408.95c5.423-7.5 5.84-17.49.417-25.41L348.724 576.624c-15.016-21.239-18.352-48.725-8.342-72.463l75.08-179.907c12.514-30.401 53.391-35.815 73.829-9.995l374.566 478.92a21.557 21.557 0 0 0 34.204 0l493.029-634.673c27.94-36.231 81.75-39.146 113.87-6.247l150.16 154.504c26.69 27.902 28.36 71.63 3.34 101.198Z"/></g></svg>
```

Reply:
Fixed in dd41a59e930d91a34b54bb96f3fb58fcfc76b8c0